src/imageio/imageio_im.c

```diff
@@ -133,6 +133,7 @@ dt_imageio_retval_t dt_imageio_open_im(dt_image_t *img, const char *filename, dt
   // See https://github.com/darktable-org/darktable/issues/13090 regarding the consequences.
   DestroyMagickWand(image);
 
+  img->buf_dsc.cst = IOP_CS_RGB;
```

Comment:
If I'm reading the diff correctly this is a bug fix as it was not set on `imageio.c` for ImageMagick case, right?

Reply:
Yes, you are right, in the case of the ImageMagick loader, IOP_CS_RGB was not set.